admin-dev/themes/new-theme/js/pages/order/create/create-order-map.js

```diff
@@ -129,6 +129,8 @@ export default {
   deliveryOptionSelect: '#delivery-option-select',
   totalShippingField: '.js-total-shipping-tax-inc',
   freeShippingSwitch: '.js-free-shipping-switch',
+  recycledPackagingSwitch: '.js-recycled-packaging-switch',
+  isAGiftSwitch: '.js-is-gift-switch',
```

Comment:
```suggestion
  isAGiftSwitch: '.js-is-gift-switch:checked',
```

Reply:
In `CreateOrderPage::_listenForCartEdit` we use this selector to listen for changes. If I follow your change, it won't work fully 🤔 . Clicking on the "checked" part will trigger the change but not the unchecked part, unlike current implementation.

Or I need 2 selectors.

Reply:
In this case you need two selectors :)

Reply:
Fixes using 2 selectors